### PR TITLE
add Windows support

### PR DESCRIPTION
**Pull request by gillins**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

This adds Windows support by using msys make. If this gets merged, I'll be able to submit a PR for `poppler` also.

### CODE REVIEW

**Comment by gillins** [comment]
@conda-forge-admin, please rerender

**Comment by pkgw** [comment]
Thanks! Just one question: did you check whether this could become a `noarch` package? I think I might have created this package at the time when `noarch` only worked for Python packages.

(Off the top of my head I don't know if `noarch` packages on Windows automatically install inside a subdirectory of `Library` or not ... something to check.)

**Comment by gillins** [comment]
I hadn't caught up with the fact there was now a `noarch: generic`. I'll have a look and see what it does under Windows. 

BTW we might be able to override where poppler looks for the data by patching this line (so it could be under `%PREFIX$/share` etc): https://gitlab.freedesktop.org/poppler/poppler/blob/master/CMakeLists.txt#L301

**Comment by ocefpaf** [comment]
I don't think a `noarch` package is possible here but let's ask @msarahan just to be sure.

**Comment by gillins** [comment]
@conda-forge-admin, please rerender

**Comment by gillins** [comment]
I've just pushed a change to use `noarch`. Seems to work ok. I have put a build on the "gillins" channel for testing.
I seem to be able to install this package on Windows - as suspected it puts the files under `%PREFIX%`rather than `%LIBRARY_PREFIX%` but does this matter? I think we can patch `poppler` to look here.
Certainly makes it nice and easy this way.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,18 +11,18 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  skip: true  # [win]
+  number: 1
+  noarch: generic
   script:
-    - make install prefix=$PREFIX
+    - make install prefix=$PREFIX
```
Comment (pkgw) [inline recipe/meta.yaml]: It looks like this change just adds a space at the end of the line ... can you take that out?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,18 +11,18 @@ source:
...
+    - make install prefix=$PREFIX 
 
 requirements:
   build:
   run:
 
 test:
   commands:
-    - test -f $PREFIX/share/poppler/cidToUnicode/Adobe-CNS1
+    - test -f $PREFIX/share/poppler/cidToUnicode/Adobe-CNS1
```
Comment (pkgw) [inline recipe/meta.yaml]: Also here.

**Review by pkgw — COMMENTED** [review]
I'm inclined to make this package `noarch: generic` as proposed here, but it's true that it would install files in an awkward/unexpected place on Windows. @ocefpaf/@msarahan, any opinions as to how big of a deal that is?

**Comment by ocefpaf** [comment]
LGTM, what do you think @pkgw?

**Pull request merged**